sdk/core/azure_core/src/http/policies/request_instrumentation.rs

```diff
@@ -0,0 +1,716 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use crate::{
+    http::{headers, options::RequestInstrumentationOptions, Context, Request},
+    tracing::{Span, SpanKind},
+};
+use std::sync::Arc;
+use typespec_client_core::{
+    http::policies::{Policy, PolicyResult, RetryPolicyCount},
+    tracing::Attribute,
+};
+
+const AZ_NAMESPACE_ATTRIBUTE: &str = "az.namespace";
+const AZ_SCHEMA_URL_ATTRIBUTE: &str = "az.schema.url";
```

Comment:
we don't need this on Rust - there is a dedicated property on the instrumentation scope (which we pass when we create a tracer) - https://github.com/open-telemetry/opentelemetry-rust/blob/d59aded375ac82b0a6a99521a1a7973976762469/opentelemetry/src/common.rs#L474

We use this attribute on .NET where the BCL doesn't provide dedicated property - it will in ,NET 10 - https://github.com/dotnet/runtime/issues/63651

Reply:
Oh, and this is an interesting issue - we *do* need `az.schema.url` for `telemetry` spans and tracers since the `telemetry` package doesn't have a way of specifying the schema url. 

Oh well, I'll deal with that when I have to deal with that.